ssts/segmentation/example.py

```diff
@@ -76,21 +84,19 @@ def main():
         pca_components = pre_seg.pca.transform(data.reshape(n, c)) # NOTE PCA was trained while fitting pre_seg
         pca_components = pca_components.reshape(h, w, num_pca_components) # shape (512, 512, num_pca_components)
 
-        # Initialize Watershed segmentation # NOTE this module can be applied after any example
-        post_seg = seg_water.SegmenterWatershed(pers_thresh=0.3) # TODO why is it stochastic?
-
-        # Apply watershed segmentation on output of GMM segmentation
-        post_labels = post_seg.fit_transform(pre_labels)
+        # Create unique masks per grain
+        post_labels = measure.label(pre_labels, connectivity=2)
```

Comment:
This method is better at segmenting binary masks, as opposed to the watershed algorithm